zimui/src/components/TopicHome.vue

```diff
@@ -22,6 +22,9 @@ const topic = ref<Topic>()
 const dataLoaded = ref(false)
 const $loading = useLoading()
 
+const errMessage = computed(() => main.errorMessage)
```

Comment:
No need for this computed value, please use the store value directly to make things clearer